handy.h

```diff
@@ -108,6 +108,34 @@ blindly casts away const.
 #define MUTABLE_IO(p)	((IO *)MUTABLE_PTR(p))
 #define MUTABLE_SV(p)	((SV *)MUTABLE_PTR(p))
 
+/*
+=for apidoc_section $SV
+=for apidoc   Am |AV *|AV_FROM_REF|SV * ref
+=for apidoc_item |CV *|CV_FROM_REF|SV * ref
+=for apidoc_item |HV *|HV_FROM_REF|SV * ref
+
+The C<I<*>V_FROM_REF> macros extract the C<SvRV()> from a given reference SV
+and return a suitably-cast to pointer to referred SV. When running under
+C<-DDEBUGGING> assertions are also applied that check that I<ref> is
```

Comment:
I realize that the modern style is to drop a lot of commas, but my training was to separate clauses by commas.  The 'When...GGING>" is a clause that I would put a comma after